docs/upgrade/0.3.5-to-0.3.6.rst

```diff
@@ -0,0 +1,112 @@
+Upgrade from 0.3.5 to 0.3.6
+============================
+
+SecureDrop 0.3.6 was a maintenance release to update the expiration
+date on the GPG signing key used for the Freedom of the Press
+apt repository. The signing key expired on October 26, 2015, preventing
+automatic updates from Freedom of the Press until the issue is
+manually resolved.
+
+
+.. note:: We're considering creating a separate ``securedrop-keyring`` 
```

Comment:
@ageis Great catch, I've removed the specific name in d2212a2. Off the cuff I like the sound of `securedrop-keyring`, but we can discuss pros and cons in a separate issue.